SpywareFilter/sections/mobile.txt

```diff
@@ -84,7 +96,6 @@
 ||mobileapptracking.com^
 ||mobile.bigfishgames.com/telemetry/
 ||ilyonanalytics.herokuapp.com^
-||register.appsflyer.com^
 ||kaffnet.com^
```

Comment:
this is mobile ad server
||ad.api.kaffnet.com^
||ad.adapter.kaffnet.com^
||kaffnet.com^
||sdk.api.kaffnet.com^